api/jobs/batch.py

```diff
@@ -206,6 +209,25 @@ def cancel(batch_job):
     update(batch_job['_id'], {'state': 'cancelled'})
     return cancelled_jobs
 
+def check_state(batch_id):
+    """
+    Return True if all jobs are complete or failed
```

Comment:
The description mentions it's a True/False response but it looks like it returns a state below.